catalogue/webapp/components/SearchNoResults/SearchNoResults.tsx

```diff
@@ -5,7 +5,7 @@ import { FunctionComponent } from 'react';
 import { PaletteColor } from '@weco/common/views/themes/config';
 
 type Props = {
-  query: string;
+  query: string | undefined;
```

Comment:
As it would currently just mean an empty component (but we'd still like some "spacing" between the nav and the footer), maybe something to look at integrating within the `SearchPageLayout` file instead.